pyaedt/modeler/modeler3d.py

```diff
@@ -115,20 +115,19 @@ def create_3dcomponent(
             Edit password.
             The default value is an empty string.
         password_type : str, optional
-            Password type. Value can either be ``UserSuppliedPassword`` or ``InternalPassword``.
-            The default value is ``UserSuppliedPassword``.
-        hide_contents : bool, optionale
-            Whether to hide contents.
-            The default is ``False``.
+            Password type. Options are ``UserSuppliedPassword`` and ``InternalPassword``.
+            The default is ``UserSuppliedPassword``.
+        hide_contents : bool, optional
+            Whether to hide contents. The default is ``False``.
         replace_names : bool, optional
-            Whether to replace objects and materials names.
+            Whether to replace objects and material names.
             The default is ``False``.
         component_outline : str, optional
             Component outline. Value can either be ``BoundingBox`` or ``None``.
             The default is ``BoundingBox``.
         auxiliary_dict_file : bool or str, optional
-            Whether to export or not the auxiliary file containing information about defined datasets and Icepak monitor
-             objects. A destination file can be specified using a string.
+            Whether to export the auxiliary file containing information about defined datasets and Icepak monitor
+            objects. A destination file can be specified using a string.
```

Comment:
Do the line breaks did to be adjusted here?

Reply:
No, it's fine like that, thanks!